server/controllers/food.js

```diff
@@ -160,9 +159,14 @@ export default {
   }
 }
 
+function isAdmin(role) {
+  return role === ADMIN_ROLE
+}
+
 function authorizeByRole(userRoles, roles = []) {
-  if (!intersection(userRoles, [...roles, ADMIN_ROLE]).length) {
-    throw new ForbiddenError
```

Comment:
@kenjiO I think this should be changed to UnathorizedError

Reply:
If there is a user but they don't have the required roles it should be forbidden, if there's no user it should be unauthorized. The `requiresLogin` middleware used in the routers handles throwing unauthorized errors

Reply:
ok, thanks